sos_brazil/exceptions.py

```diff
@@ -75,3 +75,18 @@ def __init__(
         super().__init__(*args, **kwargs)
         self.detail = {key.lower(): message}
         self.status_code = status_code
+
+
+class MinimumAdminValue(APIException):
```

Comment:
To maintain a pattern, implement Exception, like `MinimumAdminValueException`

Reply:
Done, my Leader